README.md

```diff
@@ -1,47 +1,56 @@
 # containerd API Python package
 
-This repo provides a Python3 API to [containerd's](https://containerd.io) (gRPC)
-API, directly generated from the [original containerd `.proto` API
+This repository provides a Python3 API to [containerd's](https://containerd.io)
+(gRPC) API, directly generated from the [original containerd `.proto` API
 definitions](https://github.com/containerd/containerd/tree/master/api). As it is
 generated from the protocol files, this Python package does not aim to be a
 fully Pythonesque package. In consequence, the usual idiosyncrasies of gRPC and
 protoc shine through.
 
-> **Note:** with Python2 going end-of-life in January 2020 we don't intend to
-> support Python2 in this package at this very late time in the lifecycle.
+> **Note:** with Python2 going end-of-life in January 2020 we don't support
+> Python2 in this package at this very late time in the lifecycle.
 
 ## Versioning
 
-The versioning of this package complies with [PEP 440](https://www.python.org/dev/peps/pep-0440/).
+The versioning of this package complies with [PEP
+440](https://www.python.org/dev/peps/pep-0440/).
 
-The version is composed of the version of the supported containerd API (e.g. 1.2 or 1.3) and an incremental number for each pycontainerd release for that specific containerd API version (starting from 0) connected with a '.' (a dot).
+The version is composed of the version of the supported containerd API (e.g. 1.2
+or 1.3) and an incremental number for each pycontainerd release for that
+specific containerd API version (starting from 0) connected with a '.' (a dot).
 
-Ideally the Python containerd API has to be generated only once per containerd API version, resulting in x.y.0 package versions.
+Ideally the Python containerd API has to be generated only once per containerd
+API version, resulting in x.y.0 package versions.
 
 The result is version numbers like:
- * 1.2.1 for the second release for API 1.2
- * 1.3.0 for the first release for API 1.3
+
+- 1.2.1 for the second release for API 1.2
+- 1.3.0 for the first release for API 1.3
 
 ## License
 
-This project is licensed as [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) (SPDX-License-Identifier: Apache-2.0).
+This project is licensed as [Apache License, Version
```

Comment:
Although having a line break inside of a MarkDown hyperlink parentheses (text or URL) [should work](https://stackoverflow.com/questions/19985235/break-long-lines-in-markdown-code), if it's your IDE automatically doing so, please take care not to create line breaks in the wrong place (between `]` and `(`).

Reply:
I'll clean this up too, maybe I can also find a way to keep the markdown reformatter plugin in vsc from stumping on links.